src/definitions/behaviors/form.js

```diff
@@ -1207,13 +1207,7 @@ $.fn.form = function(parameters) {
               module.debug('Using field name as identifier', identifier);
               field.identifier = identifier;
             }
-            var isDisabled = true;
-            $.each($field, function(){
-                if(!$(this).prop('disabled')) {
-                  isDisabled = false;
-                  return false;
-                }
-            });
+            var isDisabled = $field.is(':disabled');
```

Comment:
This line doesn't work properly when a field has multiple input. Example:
```html
<fieldset class="disabled grouped fields" disabled>
  <label>Bananaaaaa</label>
  <div class="field">
    <div class="ui checkbox">
      <input type="checkbox" name="fruit" tabindex="0" class="hidden">
      <label>Apples</label>
    </div>
  </div>
  <div class="field">
    <div class="ui checkbox checkbox">
      <input type="checkbox" name="fruit" tabindex="0" class="hidden">
      <label>Oranges</label>
    </div>
  </div>
  <div class="field">
    <div class="ui checkbox checkbox">
      <input type="checkbox" name="fruit" tabindex="0" class="hidden">
      <label>Pears</label>
    </div>
  </div>
  <div class="field">
    <div class="ui checkbox checkbox">
      <input type="checkbox" name="fruit" tabindex="0" class="hidden">
      <label>Grapefruit</label>
    </div>
  </div>
</fieldset>
```

Reply:
Yes,  we added that field loop because otherwise radio groups were not valiadated if the first one was disabled. https://github.com/fomantic/Fomantic-UI/pull/372
So there was a reason why each field has to be checked separately

Reply:
@prudho i still don't understand what's your "work" means. all fields inside `fieldset[disabled]` are disabled [by design](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#attr-disabled). jquery [`:disabled` selector](https://api.jquery.com/disabled-selector/) works properly [even in IE](https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled#Browser_compatibility), while `prop('disabled')` cannot handle this case.

@lubber-de updated PR and added your test case in my demo

Reply:
@jameschenjav sorry i was mistaken there :/ I supposed that validation was testing an eventual `disabled` class for field groups, which isn't the case here.